.github/workflows/nodejs.yaml

```diff
@@ -0,0 +1,50 @@
+name: Installing dependencies
+'on':
+  workflow_dispatch: null
+  push:
+    branches:
+      - main
+      - develop
+    tags:
+      - v*
+jobs:
+  nodejs:
+    name: Tests of releases NodeJS
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 10.x
+          - 12.x
+          - 14.x
+          - 15.x
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Install nodejs dependencies ${{ matrix.node-version }}'''
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ matrix.node-version }}'
+      - run: |
```

Comment:
We should cache NPM dependencies. Maybe se can use https://github.com/bahmutov/npm-install instead of traditional `npm install`.